### PR TITLE
Dropdown fix

### PR DESCRIPTION
**Pull request by AquiGorka**
cc @deamme

### CODE REVIEW

File: src/components/DropDown/DropDown.js [hunk 1]
```diff
@@ -47,8 +47,8 @@ function useDropDown({
 
   useEffect(() => {
     if (selected === -1 || !items[selected]) {
-      if (label) {
-        setSelectedLabel(label)
+      if (placeholder || label) {
```
Comment (bpierre) [inline src/components/DropDown/DropDown.js]: Shouldn’t `label` take priority over `placeholder`?

Comment (bpierre) [inline src/components/DropDown/DropDown.js]: `placeholder` should be added to the dependencies too :-)

Comment (AquiGorka) [reply src/components/DropDown/DropDown.js]: `label` is deprecated, I think it will eventually fade away...

Comment (AquiGorka) [reply src/components/DropDown/DropDown.js]: @deamme I think you did this on your PR no?

Comment (bpierre) [reply src/components/DropDown/DropDown.js]: Oh right! I was mistaking with `renderLabel`. All good then :-)

Comment (bpierre) [reply src/components/DropDown/DropDown.js]: No it’s not there (`placeholder` wasn’t in the `useEffect()` before) => https://github.com/aragon/aragon-ui/blob/newstyle-0.8/dropdown-fix/src/components/DropDown/DropDown.js#L56

Comment (AquiGorka) [reply src/components/DropDown/DropDown.js]: ahhhh you meant the effect deps, jajaja 😄 sure

**Pull request merged**